src/index.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { readFileSync } from "fs";
+import { resolve } from "path";
+import { TypeScriptSWCLoader } from "./index";
+
+describe("TypeScriptSWCLoader", () => {
```

Comment:
maybe we should also have a test suite that uses cosmiconfig package and test the loader with it?

Reply:
done: ea930c9